Storage/Site.swift

```diff
@@ -85,4 +85,11 @@ public class Site: Identifiable {
         self.title = title
         self.bookmarked = bookmarked
     }
+
+    // Helper property to quickly get the favicon url
+    public var faviconURL: NSURL? {
+        return icon?.url != nil ? NSURL(string: (icon?.url)!) : nil
```

Comment:
I don't think this method is necessary. I think there's value in keeping the `Site` separate from the `Favicon` and if you need the favicon URL you can just use `site.favicon.url`. I just don't want to bind a Site to a favicon since they are two separate things. In the future we might have the tendency to ignore the `Favicon` property and just use this instead.